rosbag2_cpp/test/rosbag2_cpp/test_time_controller_clock.cpp

```diff
@@ -200,3 +200,31 @@ TEST_F(TimeControllerClockTest, resumes_at_correct_ros_time)
   clock.resume();
   EXPECT_EQ(clock.now(), RCUTILS_S_TO_NS(1));
 }
+
+TEST_F(TimeControllerClockTest, change_rate)
+{
+  rosbag2_cpp::TimeControllerClock clock(ros_start_time, 1.0, now_fn);
+  rcutils_time_point_value_t expected_ros_time = 0;
+
+  // 1 second passes for both
+  return_time += std::chrono::seconds(1);
```

Comment:
Is `return_time` used anywhere?

Reply:
yes, `return_time` is the value returned by the mocked `now_fn` that we give to the clock. (https://github.com/ros2/rosbag2/pull/727/files#diff-ea9ff4b69cd29013d4599d8e20313ab1f1dd918e82e810755dad97abfd9f4a51R32) - this is how I am able to control the "flow of steady time" in the tests to get specific scenarios.

Reply:
That makes sense, thanks!